pkg/executables/cmk.go

```diff
@@ -471,6 +506,11 @@ type cmkNetwork struct {
 	Name string `json:"name"`
 }
 
+type cmkVirtualMachine struct {
```

Comment:
cmkDomain and cmkVirtualMachine are identical. How about introducing cmkResourceIdentifier with Id and Name props and using it for both?

Reply:
makes sense